node/src/BaseClient.ts

```diff
@@ -482,6 +483,26 @@ export class BaseClient {
         return this.createWritePromise(createGet(key));
     }
 
+    /**
+     * Gets a string value associated with the given `key`and deletes the key.
+     *
+     * See https://valkey.io/commands/getdel/ for details.
+     *
+     * @param key - The key to retrieve from the database.
+     * @returns - If `key` exists, returns the `value` of `key`. Otherwise, return `null`.
```

Comment:
No dash in returns section
```suggestion
     * @returns If `key` exists, returns the `value` of `key`. Otherwise, return `null`.
```